src/linked_list.py

```diff
@@ -0,0 +1,79 @@
+"""The Linked List Module Creates a LinkedList."""
+
+
+class Node(object):
+    """Defining Node Class."""
+
+    def __init__(self, val, next=None):
+        """Instantiate Node Class."""
+        self.val = val
+        self.next = next
+
+
+class LinkedList(object):
+    """Defining class of LinkedList."""
```

Comment:
When writing doc strings for classes, it's a good idea to include in the doc string the methods of the class object and what they do.